st/clitests/awss3api_spec.py

```diff
@@ -1136,6 +1136,208 @@ def get_md5_sum(fname):
 AwsTest('Aws can delete destination bucket').delete_bucket("destination-seagatebucket")\
     .execute_test().command_is_successful()
 
+# **********************************************************************************************
+# ************ Multipart UploadPartCopy API Support ********************************************
+
+# 0. Create Source Bucket
+AwsTest('Aws can create source bucket')\
+    .create_bucket("mpcp-src-bucket")\
+    .execute_test().command_is_successful()
+
+# 1. Upload simple objects in source objects.
+AwsTest('Aws can put first simple object')\
+    .put_object("mpcp-src-bucket", "simpleobj1", 5242880)\
+    .execute_test().command_is_successful()
+AwsTest('Aws can put second simple object')\
+    .put_object("mpcp-src-bucket", "simpleobj2", 5242880)\
+    .execute_test().command_is_successful()
+AwsTest('Aws can put third simple object')\
+    .put_object("mpcp-src-bucket", "simpleobj3", 1048576)\
+    .execute_test().command_is_successful()
+
+# 2. Create Destination Bucket
+AwsTest('Aws can create destination bucket')\
+    .create_bucket("mpcp-dest-bucket")\
+    .execute_test().command_is_successful()
+
+# 3. Create multipart upload
+result = AwsTest('Aws can create multipart session')\
+            .create_multipart_upload("mpcp-dest-bucket", "mpobject",
+                11534336, "test=uploadpartcopytest")\
+            .execute_test().command_is_successful()
+upload_id = get_upload_id(result.status.stdout)
+print("Upload Id: ", upload_id)
+
+# 4. Copy simple objects as seperate parts in multipart uplaod session. 
+result = AwsTest('Aws can copy object as first part')\
+            .upload_part_copy("mpcp-dest-bucket", "mpobject", "1" ,
+                              upload_id, "mpcp-src-bucket/simpleobj1")\
+            .execute_test().command_is_successful()
+e_tag_1 = result.status.stdout
+if '"' in e_tag_1:
```

Comment:
if e_tag is blank, the same blank value will be provided to complete-multipart-upload and the test will fail. 
so that is the expected result I gues..

Reply:
> if '"' in e_tag_1:

ok its 'in' and not is. Its fine then